src/app/helptext/system/certificates.ts

```diff
@@ -24,6 +24,23 @@ export const helptext_system_certificates = {
       validation: [Validators.required]
     },
 
+    key_type: {
+      placeholder: T("Key Type"),
+      tooltip: T(
+        'See <a href="https://crypto.stackexchange.com/questions/1190/why-is-elliptic-curve-cryptography-not-widely-used-compared-to-rsa"></a>\
```

Comment:
Can't we just reuse the help string variable from above, rather than duplicating the contents?